lib/printers/json.js

```diff
@@ -15,8 +15,12 @@ class JsonPrinter {
       let warnings = this.options.quiet
         ? []
         : fileErrors.filter((error) => error.severity === Linter.WARNING_SEVERITY);
+      let todos =
+        this.options.quiet || !this.options.includeTodo
```

Comment:
So if we have `includeTodo` set to `true` _and_ `quiet`, this is going to remove the todos from output. Do we want that? If you set `includeTodo`, I feel like you're explicitly asking to see the todos.

Reply:
The same can be said to `quiet` since you're explicitly asking for the output to be quiet, so it's matter of deciding which one has a higher priority. I thought `quiet` should be the overriding setting here due to the existing behavior for warnings: include warnings but if quiet is passed make it output nothing. Thoughts?

Reply:
Ya that makes sense.